controls/radgauge/getting-started/creating-linear-gauge.md

```diff
@@ -85,17 +79,17 @@ The __LinearScale__ control allows you to display different types of indicators
 	</telerik:RadVerticalLinearGauge>
 {{endregion}}
 
-Here is a snapshot of the result.
+#### __Image 2: Result from Example 2__
 
-![](images/RadGauge_GettingStarted_LinearGauge_03.png)
+![RadVerticalLinearGauge with Indicator](images/RadGauge_GettingStarted_LinearGauge_03.png)
 
 ## Set Indicator's Position
 
-To make the indicator point to a certain value off the scale, you have to use the Value property of the indicator. Here is an example:
+To make the indicator point to a certain value on the scale, you have to use the Value property of the indicator. Here is an example:

```

Comment:
To make the indicator point to a certain value on the scale, you have to use the __Value__ property of the indicator. ~~Here is an example:~~